source/content/wordpress-cron.md

```diff
@@ -71,62 +86,79 @@ terminus wp <site>.<env> -- cron <your wp-cron command and switches here>
 
-[WPX Cron Manager Lite](https://wordpress.org/plugins/wpx-cron-manager-light/ "WPX Cron Manager Lite") will do a similar job with a slightly different UI. This plugin requires you to do a one-time installation of the WPX framework, which you can do straight from the plugin manager page.
+[WPX Cron Manager Lite](https://wordpress.org/plugins/wpx-cron-manager-light/ "WPX Cron Manager Lite") works similarly with a slightly different UI. This plugin requires you to do a one-time installation of the WPX framework, which you can do straight from the plugin manager page.
 
 ## Manage WP-Cron Externally
 
-If you're looking for more control over your site's cron jobs, or you don't want WP-Cron to handle tasks internally, you can use external crons instead. This will solve the problems discussed above for high traffic and low traffic sites.
+You can use external Crons if you want more control over your site's Cron jobs, or if you don't want WP-Cron to handle jobs internally. This will solve the problems with high traffic and low traffic sites discussed in the Troubleshooting section.
 
 ### Disable WP-Cron
 
-The first thing you'll need to do is disable WP-Cron's internal processing. Add the following line to your `wp-config.php` file:
+Disable WP-Cron's internal processing by adding following code to your `wp-config.php` file:
 
 ```php:title=wp-config.php
 define('DISABLE_WP_CRON', true);
 ```
 
 <Alert title="Note" type="info">
-There are many important tasks that WP-Cron takes care of, so be prepared to complete all the steps below.
+
+WP-Cron executes many important jobs. Be sure to complete all the steps below.
+
 </Alert>
 
 ### Free Services
 
-Once you have disabled WP-Cron, you will need a service that calls a URL at regular intervals. The easiest way to do this is to set up an account with a free cron service:
+You will need a service that calls a URL at regular intervals after disabling WP-Cron. The easiest way to do this is to set up an account with a free cron service:
 
 - [EasyCron](https://www.easycron.com/)
 - [Set Cron Job](https://www.setcronjob.com/)
 - [cron-job](https://cron-job.org/en/)
 
-Any of the above services will get the job done. By disabling WP-Cron, you have turned off the automatic checking and calling of the `wp-cron.php` script. You will now have to call that URL yourself using one of the services above. With most of them, it is as easy as this:
+Any of the above services will get the job done. Disabling WP-Cron turns off the automatic checking and calling of the `wp-cron.php` script. You will now have to call that URL yourself using one of the services above. Most services only require the following setup:
```

Comment:
Same as the note above.

Reply:
@CdrMarks This is a subsection of a section focused on Managing Cron Externally.